mlflow/tracking/fluent.py

```diff
@@ -2022,14 +2023,16 @@ def create_logged_model(
     if source_run_id is None and (run := active_run()):
         source_run_id = run.info.run_id
     experiment_id = experiment_id if experiment_id is not None else _get_experiment_id()
-    return MlflowClient().create_logged_model(
+    model = MlflowClient().create_logged_model(
         experiment_id=experiment_id,
         name=name,
         source_run_id=source_run_id,
         tags=tags,
         params=params,
         model_type=model_type,
     )
+    _last_logged_model_id.set(model.model_id)
+    return model
```

Comment:
this doesn't seem to work when a logged model is created by `create_logged_model`.

Reply:
Yep! We need to fix this

Reply:
This is already fixed.